src/components/ConfirmationModal.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div
+    class="absolute w-full h-full bg-black/50 flex justify-center items-center"
+  >
+    <div
+      class="bg-white px-[1.5rem] lg:px-[3rem] py-[2rem] lg:py-[4rem] rounded-lg flex justify-center items-center flex-col"
+    >
+      <h1 class="text-lg lg:text-2xl font-bold text-blue">{{ heading }}</h1>
+      <p class="mt-4 text-lg">{{ para }}</p>
+      <button
+        @click="$emit('confirmButtonHandler')"
```

Comment:
do you think, the handler in the function name is important? @aayushbatra02

Reply:
no sir i will change it